template/changelog/CHANGELOG.tmpl

```diff
@@ -3,7 +3,7 @@
 {{ if gt (len $allVersions) 1 -}}
 # Changelog for [{{ .Version }}] ({{ .Date }})
 
-The following sections list the changes in ownCloud`}}{{ Name }}{{`{{ .Version }}.
+The following sections list the changes in `}}{{ Name }} {{`{{ .Version }}.
```

Comment:
This looks wrong

Reply:
Can you specify how wrong?
go Variables that should not be parsed by boilr needs to be escaped.

Reply:
But escaping by an closing tag?

Reply:
looks, wrong, but the opening tag is there